bridges/SteamBridge.php

```diff
@@ -8,45 +8,10 @@ class SteamBridge extends BridgeAbstract {
 	const MAINTAINER = 'jacknumber';
 	const PARAMETERS = array(
 		'Wishlist' => array(
-			'username' => array(
-				'name' => 'Username',
+			'userid' => array(
+				'name' => 'Steamid64 (find it on steamid.io)',
```

Comment:
>The request needs an id and not a username anymore, easy to find with steamid.io or steamidfinder.com.

This should probably also be mentioned in the tool tip.
```suggestion
				'name' => 'Steamid64 (find it on steamid.io)',
				'title' => 'User ID (17 digits). Find your user ID with steamid.io or steamidfinder.com',
```